_posts/2018-06-18-On-Obsessive-Statelessness.markdown

```diff
@@ -125,11 +125,11 @@ what's happening on their installations inside peril.
 
 <img src="/images/peril-state/peril-dashboard.png">
 
-I know, a beauty right? Taking ideas from Ashkan's [post on JWT's][jwts] I explored using JWTs to a few user-y,
+I know, a beauty right? Taking ideas from Ashkan's [post on JWT's][jwts] I explored using JWTs to fix a few user-y,
 database-y related problems.
 
 A JWT is a string made of three components, a header which says how it is signed, a base64 chunk of JSON data and a
-signature verifying the data. You can always read the data inside a JWT, but you need to know the public key used to
+signature verifying the data. You can always read the data inside a JWT, but you need to know the private key used to
```

Comment:
I'm pretty sure this should be the private key, no?

Reply:
I don't understand this, but that's ok! Pushed a new commit so this should be ready to merge on green.

Reply:
Hah, it's exactly the same as how you can verify that someone has signed a document with their public key, instead of their private key - I found this explanation to be pretty reasonable https://www.youtube.com/watch?v=YEBfamv-_do